ast/ast.go

```diff
@@ -192,6 +198,13 @@ func NewSubGraph(id, l Attrib) (*SubGraph, error) {
 	return g, nil
 }
 
+func randInt63() int64 {
+	randLock.Lock()
+	result := r.int63()
+	randLock.Unlock()
```

Comment:
Super duper nit.  I think we can use defer, the compiler optimizes these small cases.

Isn't it `r.Int63()` with a capital letter?

Reply:
Just tried, here 1.11 doesn't optimize defer. Also defer prevents inlining.

Reply:
Thank you.  I learned something :)